seed/challenges/02-javascript-algorithms-and-data-structures/basic-javascript.json

```diff
@@ -1900,7 +1900,9 @@
       ],
       "tests": [
         "assert((function(){if(typeof myArray != 'undefined' && myArray[0] == 45 && myArray[1] == 64 && myArray[2] == 99){return true;}else{return false;}})(), 'message: <code>myArray</code> should now be [45,64,99].');",
-        "assert((function(){if(code.match(/myArray\\[0\\]\\s*=\\s*/g)){return true;}else{return false;}})(), 'message: You should be using correct index to modify the value in <code>myArray</code>.');"
+        "assert((function(){if(code.match(/myArray\\[0\\]\\s*=\\s*/g)){return true;}else{return false;}})(), 'message: You should be using correct index to modify the value in <code>myArray</code>.');",
+        "assert(code.match(/var\\s+myArray\\s*=\\s*\\[\\s*18\\s*,\\s*64\\s*,\\s*99\\s*\\]/g), 'message: The initial <code>myArray</code> line should not be changed.');",
```

Comment:
Have you gotten this test to correctly work @MiloATH? When I change the original array (per the issue thread) it still passes this test. I wonder if it's something to do with the original array still being in the `tail` section of the code.